sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/sampleoverview/SampleOverviewViewModel.groovy

```diff
@@ -0,0 +1,19 @@
+package life.qbic.portal.sampletracking.components.sampleoverview
+
+import life.qbic.business.project.Project
+import life.qbic.business.samples.Sample
+
+/**
+ * <b><short description></b>
+ *
+ * <p><detailed description></p>
+ *
+ * @since <version tag>
```

Comment:
here too

Reply:
as this class is no longer present. I think this comment can be resolved.